etc/kayobe/ansible/build-ofed-rocky.yml

```diff
@@ -42,7 +42,8 @@
 
     - name: Add DOCA host repository package
       ansible.builtin.dnf:
-        name: https://developer.nvidia.com/downloads/networking/secure/doca-sdk/DOCA_2.8/doca-host-2.8.0-204000_{{ stackhpc_pulp_doca_ofed_version }}_rhel9{{ stackhpc_pulp_repo_rocky_9_minor_version }}.x86_64.rpm
+        name: "https://developer.nvidia.com/downloads/networking/secure/doca-sdk/DOCA_2.8/doca-host-2.8.0-204000_\
```

Comment:
Came here because I thought it was going to be a small one, turned out to a big change, so will need to review at some other time! One comment though ... I don't like the line length rule as in my opinion it makes thing less readable and less grep-able. I'm thinking in particular when we are escaping new lines like this. Thoughts?

Reply:
Yeah, I don't really like them either. I'm happy to revert those changes and ignore the rule

Reply:
Done